test/suite/test_bug036.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
You also have branch POC-control-points-WT-10905-reproducer which does not have a PR.
Is that branch obsolete? Should it be deleted?

Reply:
yes the branch is obsolete now, and can be deleted.